### PR TITLE
Overview update to the readme file.

### PR DESCRIPTION
**Pull request by MikaylaRay44**
Hey,so I know how crazy busy you might be so I went ahead and did the request I made on Facebook. I know a whole separate branch might be a bit much for a readme update but if there is anything that needs to be changed or if it is going to be an ongoing change I figured it was best to keep this discussion and code separate.

### CODE REVIEW

**Comment by alis0nc** [comment]
No, a feature branch is always a good thing. And I like the little more explanation you've given in the readme. There's a few issues, though, that mean that i'm not going to merge this immediately. 

* The first line heading should just be `kuthesis`, since that's the name of the class.
* It's just one LaTeX class, but with options that make it apply to all four kinds of theses. I'd rephrase the description line as "LaTeX *class* for the four different kinds of Kettering University theses."
* The link to cite "What Is LaTeX" should have http:// prefix.

Change these things, and you're as good as merged. Thanks for contributing!

**Comment by MikaylaRay44** [comment]
Alright then, all changes make sense to me and were made.

**Comment by alis0nc** [comment]
Merged! Thanks!
